src/consumer_api_socket_endpoint.ts

```diff
@@ -1,65 +1,46 @@
 import {Logger} from 'loggerhythm';
 
-import {UnauthorizedError} from '@essential-projects/errors_ts';
 import {IEventAggregator, Subscription} from '@essential-projects/event_aggregator_contracts';
 import {BaseSocketEndpoint} from '@essential-projects/http_node';
-import {IIdentity, IIdentityService} from '@essential-projects/iam_contracts';
+import {IIdentity} from '@essential-projects/iam_contracts';
+import {IEndpointSocketScope, ISocketClient} from '@essential-projects/websocket_contracts';
 
 import {IConsumerApi, Messages, socketSettings} from '@process-engine/consumer_api_contracts';
 
 const logger: Logger = Logger.createLogger('consumer_api:http:socket.io_endpoint');
 
-type UserSubscriptionDictionary = {[userId: string]: Array<Subscription>};
+type UserSubscriptionDictionary = { [userId: string]: Array<Subscription> };
```

Comment:
Previous version was better.

```suggestion
type UserSubscriptionDictionary = {[userId: string]: Array<Subscription>};
```